modules/ols-about-running-openshift-lightspeed-in-disconnected-mode.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/about/ols-about-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-running-openshift-lightspeed-in-disconnected-mode_{context}"]
+= About running {ols-long} in disconnected mode
+
+The {ols-long} Operator and the {ols-long} service can work in a disconnected environment. A disconnected environment is an environment that does not have full access to the internet. 
+
+In a disconnected environment, you must mirror the required container images into the environment. For more information, see "Mirroring in disconnected environments" in the {ocp-product-title} product documentation.
```

Comment:
You don't want to give a hard link here? I would. Perhaps as an improvement later on. Not a merge blocker.